src/Evolution/DiscontinuousGalerkin/Limiters/WenoOscillationIndicator.cpp

```diff
@@ -163,6 +165,54 @@ Matrix compute_indicator_matrix(
   return result;
 }
 
+// Helper function for caching a matrix that depends on an Index.
+// The helper handles converting from an Index (typically obtained from a Mesh)
+// to a series of integers (used to call into the StaticCache) and back to an
+// Index (used to compute the matrix).
```

Comment:
Isn't the important thing this function does the static-caching, not the converting of indices (which is just an implementation detail)?

Reply:
The function does both, and both are stated in this documentation. I don't think I'm claiming one is more important. The function is in an unnamed namespace because the whole thing is an implementation detail. Is there a change you'd like me to make?

Reply:
I think just this change https://github.com/sxs-collaboration/spectre/pull/2857#discussion_r576252990 would suffice, since the wording "index" I find a bit confusing